src/components/sbb-card/readme.md

```diff
@@ -1,49 +1,78 @@
-The `sbb-card` component is a generic content container as an action element; its task is to contain HTML elements related to a single subject.
+The `sbb-card` component is a generic content container; its task is to contain content related to a single subject. 
+There are various sizes (affecting paddings) and colors available.
 
-Internally, it is either rendered as an anchor (`<a>`, when used with a `href` attribute) or as a button (when used without a `href` attribute).
-When rendered as a button, it is possible to listen to the click event to add behavior.
+The `sbb-card-badge` component can be used via slot to display a badge in the upper right corner.
+The badge is hidden with card sizes `xs` or `s`.
 
-There are six size variants available (from `xs` to `xxl`), which can be configured via the `size` attribute value,
-and two background variants, white by default and milk when the `color` attribute is set to `milk`.
-Set the `active` attribute to display the card in an active state, which is bordered and has no hover effect.
+## Action
+
+To add an action to a card, add a `<sbb-card-action>` to the main slot. With the `<sbb-card-action>` 
+all the card area becomes clickable. 
+Consult [API docs of `sbb-card-action`](/docs/components-sbb-card-sbb-card-action--docs) for detailed information.
```

Comment:
There is not a lot of information in the action documentation? Is that intentional?
If so, the phrasing could maybe be adapted?

Reply:
What I meant is that the api documentation of props is there. Do you have a suggestion how to write it best?